<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>328</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1000</width>
    <height>750</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>JPE Manual Control</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="styleSheet">
    <string notr="true">#centralwidget{background-color: rgb(0,0,0);}

QLabel{
color:rgb(168,168,168)
}

QLineEdit{
border: 1px solid rgb(100,100,100);
background-color:rgb(230,230,230);
}

QTextEdit{
border: 1px solid black;
background-color: black;
color:rgb(168,168,168);
}

</string>
   </property>
   <widget class="QPushButton" name="push_Servers">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>15</width>
      <height>15</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_Servers{
background: rgb(144, 140, 9);
 border-radius: 4px;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_12">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>110</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Rel. Size:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_21">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>45</y>
      <width>111</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Module:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Temperature">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>45</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>293</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_14">
    <property name="geometry">
     <rect>
      <x>155</x>
      <y>45</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Temp. (K):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_8">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>80</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_title{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="text">
     <string>Step Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_JPE_Approach_Freq">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>600</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_title_5">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>15</y>
      <width>311</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">#label_title{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="text">
     <string>General Settings</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="Line" name="line_4">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>140</y>
      <width>460</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_4{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_Tip_Height">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>45</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>24m</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_11">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>110</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Num Steps:</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox_JPE_Address">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>45</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <item>
     <property name="text">
      <string>1</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>3</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>4</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>5</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>6</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="lineEdit_JPE_Approach_Steps">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="toolTip">
     <string>One step at 100% step size might correspond to approximately 25 nm. </string>
    </property>
    <property name="text">
     <string>1k</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_JPE_Approach_Size">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>110</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_10">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>110</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Freq. (Hz):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="label_13">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>45</y>
      <width>61</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>Height (m):</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="push_moveNegX">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>228</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_moveNegX{
image:url(:/nSOTScanner/Pictures/doublearrow_left.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_moveNegX{
image:url(:/nSOTScanner/Pictures/doublearrow_left.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_movePosY">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>173</y>
      <width>31</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_movePosY{
image:url(:/nSOTScanner/Pictures/doublearrow_up.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_movePosY{
image:url(:/nSOTScanner/Pictures/doublearrow_up.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_moveNegY">
    <property name="geometry">
     <rect>
      <x>106</x>
      <y>269</y>
      <width>31</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_moveNegY{
image:url(:/nSOTScanner/Pictures/doublearrow_down.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_moveNegY{
image:url(:/nSOTScanner/Pictures/doublearrow_down.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_movePosX">
    <property name="geometry">
     <rect>
      <x>146</x>
      <y>228</y>
      <width>45</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_movePosX{
image:url(:/nSOTScanner/Pictures/doublearrow_right.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_movePosX{
image:url(:/nSOTScanner/Pictures/doublearrow_right.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_movePosZ">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>174</y>
      <width>31</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_movePosZ{
image:url(:/nSOTScanner/Pictures/doublearrow_up.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_movePosZ{
image:url(:/nSOTScanner/Pictures/doublearrow_up.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="push_moveNegZ">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>270</y>
      <width>31</width>
      <height>45</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#push_moveNegZ{
image:url(:/nSOTScanner/Pictures/doublearrow_down.png);
background: black;
border: 0px solid rgb(95,107,166);
}

QPushButton:pressed#push_moveNegZ{
image:url(:/nSOTScanner/Pictures/doublearrow_down.png);
background: rgb(95,107,166);
border: 1px solid rgb(95,107,166);
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>174</y>
      <width>151</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#frame{
image:url(:/nSOTScanner/Pictures/Tip-Sample.png);
}</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>49</x>
      <y>150</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>X-Y Control</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>315</x>
      <y>150</y>
      <width>61</width>
      <height>17</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Z Control</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>119</x>
      <y>158</y>
      <width>102</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>219</x>
      <y>159</y>
      <width>3</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_2{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>319</y>
      <width>201</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_3{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_5">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>159</y>
      <width>3</width>
      <height>162</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_5{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_6">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>158</y>
      <width>22</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_6{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_7">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>158</y>
      <width>102</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_7{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_8">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>160</y>
      <width>3</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_8{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_9">
    <property name="geometry">
     <rect>
      <x>279</x>
      <y>160</y>
      <width>3</width>
      <height>162</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_9{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_10">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>158</y>
      <width>22</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_10{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_11">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>320</y>
      <width>202</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_11{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_12">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>89</y>
      <width>85</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_12{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_13">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>91</y>
      <width>3</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_13{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_14">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>91</y>
      <width>3</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_14{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_15">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>89</y>
      <width>20</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_15{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_16">
    <property name="geometry">
     <rect>
      <x>165</x>
      <y>24</y>
      <width>315</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_16{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_17">
    <property name="geometry">
     <rect>
      <x>19</x>
      <y>25</y>
      <width>3</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_17{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_18">
    <property name="geometry">
     <rect>
      <x>479</x>
      <y>25</y>
      <width>3</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_18{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="Line" name="line_19">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>24</y>
      <width>20</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_19{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_20">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>74</y>
      <width>460</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_20{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBox_Torque">
    <property name="geometry">
     <rect>
      <x>245</x>
      <y>80</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#checkBox_Torque{
color: rgb(168,168,168)
}</string>
    </property>
    <property name="text">
     <string>High Torque Mode</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="Line" name="line_21">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>89</y>
      <width>120</width>
      <height>3</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">#line_21{
color:rgb(168,168,168)
}</string>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Plain</enum>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
  </widget>
 </widget>
 <tabstops>
  <tabstop>lineEdit_Tip_Height</tabstop>
  <tabstop>lineEdit_Temperature</tabstop>
  <tabstop>comboBox_JPE_Address</tabstop>
  <tabstop>lineEdit_JPE_Approach_Steps</tabstop>
  <tabstop>lineEdit_JPE_Approach_Size</tabstop>
  <tabstop>lineEdit_JPE_Approach_Freq</tabstop>
  <tabstop>push_movePosY</tabstop>
  <tabstop>push_moveNegX</tabstop>
  <tabstop>push_movePosX</tabstop>
  <tabstop>push_moveNegY</tabstop>
  <tabstop>push_movePosZ</tabstop>
  <tabstop>push_moveNegZ</tabstop>
  <tabstop>push_Servers</tabstop>
 </tabstops>
 <resources>
  <include location="../Resources/nSOTScannerResources.qrc"/>
 </resources>
 <connections/>
</ui>
